library_generation/generate_library.sh

```diff
@@ -224,7 +224,11 @@ if [[ "${proto_only}" == "false" ]]; then
   mv_src_files "gapic" "test" "${temp_destination_path}"
   if [ "${include_samples}" == "true" ]; then
     # move java_gapic_srcjar/samples/snippets to samples/snippets
-    mv_src_files "samples" "main" "${temp_destination_path}"
+    if [[ "${proto_path}" == "grafeas/v1" ]]; then
```

Comment:
Shall we just skip generating samples for grafeas as they are currently not included in the repo?

Reply:
If we don't want to add special case like this, we need to change `include_samples` in BUILD.